src/index.html

```diff
@@ -3,9 +3,49 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.1/font/bootstrap-icons.css">
+    <title>Vehicle Rent</title>
+    <link href="//fonts.googleapis.com/css?family=Average&subset=latin" rel="stylesheet" type="text/css">
+    <link href="//fonts.googleapis.com/css?family=Allison&subset=latin" rel="stylesheet" type="text/css">
 </head>
 <body>
-    
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container">
+            <a href="#" class="navbar-brand">
+                <img src="assets/images/logo.png" id="mainimage" alt="logo" class="d-inline-block align-text-top">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="true" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="#">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#" id="carNumber">Cars</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <main class="container">
+        <div class="row" id="itemsContainer">
+
+        </div>
+        <div id="modalContainer">
+
+        </div>
+    </main>
+    <footer class="container" id="footerid">
+        <div class="row">
+            <h5>Created by microverse under CC license</h5>
+        </div>
+    </footer>
```

Comment:
- [ ] It would be better to move the footer to the bottom of the page as the wireframe.

![footer-red](https://user-images.githubusercontent.com/79876959/143727119-a44ca335-d11a-4ca7-9cb3-39ae5a88c2cf.PNG)

Reply:
moved footer to the bottom of the page as suggested by the reviewer.